src/repositories/direct-thread.repository.ts

```diff
@@ -6,15 +6,16 @@ import Chance = require('chance');
 export class DirectThreadRepository extends Repository {
   public async broadcast(options: DirectThreadBroadcastOptions) {
     const mutationToken = new Chance().guid();
-    const recipients = options.threadIds || options.userIds
-    const ids = recipients instanceof Array ? recipients : [recipients]
+    const recipients = options.threadIds || options.userIds;
+    const recipientsType = options.userIds ? 'recipient_users' : 'thread_ids';
+    const recipientsIds = recipients instanceof Array ? recipients : [recipients];
 
     const { body } = await this.client.request.send<DirectThreadRepositoryBroadcastResponseRootObject>({
       url: `/api/v1/direct_v2/threads/broadcast/${options.item}/`,
       method: 'POST',
       form: {
         action: 'send_item',
-        [options.threadIds ? 'thread_ids' : 'recipient_users']: JSON.stringify(options.userIds ? [ids] : ids)
+        [recipientsType]: JSON.stringify(options.userIds ? [recipientsIds] : recipientsIds)
```

Comment:
Could you check `recipientsType` string instead of `options.userIds`? I think it will be more explicit, because we change field format based on the recipient type.

Reply:
Everything else is good enough to merge

Reply:
done